Software/bsw/gen/CanIf_PBcfg.c

```diff
@@ -198,7 +198,7 @@ CanIfTxPduCfgType CanIfTxPduCfgObj[TX_CAN_L_PDU_NUM] =
 #endif
 
       CanIfTxPduUserTxConfirmation ,  /*This parameter defines the name of the <User_TxConfirmation>.Currently just a stub function*/
-      CAN_NM_TX_CONFIRMATION,           /* the upper layer module to which the indication of the successfully transmitted CANTXPDUID has to be routed via <User_TxConfirmation> */
+      CAN_NM,           /* the upper layer module to which the indication of the successfully transmitted CANTXPDUID has to be routed via <User_TxConfirmation> */
```

Comment:
We should be specific as there is a Tx Confirmation and Rx indication so it should be having an indication whether it's CAN_NM_TX_CONFIRMATION or CAN_NM_RX_INDICATION
that's the reason for having this parameter like that so please use this name and again if you think there is anything to change in cfg files please request them in the cfg branches to be approved and pushed for the whole teams

Reply:
As this parameter CAN_NM is the one used in AUTOSAR SWS. But if it is better to use different formulas to differentiate between Tx Confirmation and Rx indication, then I will undo these changes and push again